srcpkgs/python3-unpaddedbase64/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-unpaddedbase64'
 pkgname=python3-unpaddedbase64
 version=1.1.0
-revision=2
+revision=3
```

Comment:
what is this revbump for?

Reply:
As noted by the commit, this package was never built for python 3.8. Rev 2 is built for python 3.6.